File Attributes with null character causes exception

### Current Behavior

We updated our application from F!F 8.03 to 9.0.1, since then we see the following error in the log:
`Caused by: org.xml.sax.SAXParseException: An invalid XML character (Unicode: 0x0) was found in the value of attribute "security.selinux" and element is "file".`

The XML snippet causing this is (extracted from the log):
`
Message[27700c9:String]: [<file name="xxx.xml" size="135463" fSize="132 KiB" canonicalName="/xxx/xxx.xml" type="file" modificationDate="2025-08-26" modificationTime="12:05:13" security.selinux="system_u:object_r:public_content_rw_t:s0 "/>]]`

In Notepad, when I check 'show all symbols' it is showing null character:
<img width="281" height="46" alt="Image" src="https://github.com/user-attachments/assets/0c04eb57-2734-4cd8-9668-131abba333ca" />

I have little knowledge of selinux, so no idea if null character should be there.
I advised AI, that came up with:

> Why Null Bytes Are Appearing
> The null bytes you're seeing in your logs are likely due to:
> 
> Storage format: When SELinux attributes are stored in the filesystem's extended attributes, they may be stored as null-terminated C strings or in a binary format that includes padding bytes.
> 
> Reading method: The Java NIO [Files.readAttributes()](vscode-file://vscode-app/c:/Users/SW1147/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-browser/workbench/workbench.html) method might be reading the raw binary data from the filesystem's extended attribute storage, which includes the null termination.
> 
> Linux filesystem implementation: Some filesystems store extended attributes with null padding or termination.

I do not know the exact server setup where this happens (at client), on a simple Rocky instance that I use myself it's not happening. Although the file has security.selinux attribute, I don't see it back in the logging.

### Expected Behavior

Possibly remove null characters, so it will run without exceptions.

### Environment Information

**FF! 9.0.1**: xxx 124
Running on **xxx** using **Apache Tomcat/10.1.39**
Java Version: **OpenJDK Runtime Environment (17.0.13+11-LTS)**
Heap size: **234.1 MiB**, total JVM memory: **380.0 MiB**
Free memory: **145.9 MiB**, max memory: **934.0 MiB**
Free disk space: **1.6 GiB**, total disk space: **18.7 GiB**
Up since: **2025-08-27 11:36:14**, timezone: **UTC**

### Steps To Reproduce

Use LocalFileSystemPipe
Have selinux enabled.

### Configuration

```xml
<pipe name="writeFooterForDownload"
                  className="org.frankframework.pipes.LocalFileSystemPipe"
                  action="append"
                  root="${download.dir}"
                  getInputFromSessionKey="FileFooter">
                <param name="filename" sessionKey="FileNameStreamingInSession"/>
                <forward name="success" path="putCanonicalNameInSessionKey"/>
            </pipe>

            <pipe name="putCanonicalNameInSessionKey"
                  className="org.frankframework.pipes.XsltPipe"
                  xpathExpression="/file/@canonicalName"
                  storeResultInSessionKey="FileDLPath">
                <forward name="success" path="checkFiscalYear_3"/>
            </pipe>
```

### Input

_No response_

### What database are you using?

_No response_

### What browsers are you seeing the problem on?

_No response_

### Relevant Log Output

```console
2025-08-26 12:05:13,989 DEBUG [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification, pipe=writeFooterForDownload} stream.Message - returning byte[] Message[528482a2:byte[]] as InputStream
2025-08-26 12:05:14,020 DEBUG [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification} processors.MonitoringPipeProcessor - Pipeline of adapter [Notification] messageId [testmessage0a809b34--54910c3d_198e5d6b5b0_-8000] is about to call pipe [putCanonicalNameInSessionKey] current result (Message) [context:
Metadata.Size: 287
value:
Message[27700c9:String]: [<file name="xxx.xml" size="135463" fSize="132 KiB" canonicalName="/xxx/xxx.xml" type="file" modificationDate="2025-08-26" modificationTime="12:05:13" security.selinux="system_u:object_r:public_content_rw_t:s0 "/>]] 
2025-08-26 12:05:14,021 DEBUG [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification, pipe=putCanonicalNameInSessionKey} jta.TransactionConnectorCoordinator - no active transaction in thread [http-nio-8080-exec-5]
2025-08-26 12:05:14,022 DEBUG [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification, pipe=putCanonicalNameInSessionKey} stream.Message - returning String Message[27700c9:String] as InputSource
2025-08-26 12:05:14,105 DEBUG [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification} processors.TransactionAttributePipeLineProcessor - setting RollBackOnly for pipeline after catching exception
2025-08-26 12:05:14,110 WARN  [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification} core.Adapter - Adapter [Notification] error processing message with messageId [testmessage0a809b34--54910c3d_198e5d6b5b0_-8000]: Pipe [putCanonicalNameInSessionKey] Exception on transforming input: Cannot transform input: (SAXParseException) line [1] column [284]: An invalid XML character (Unicode: 0x0) was found in the value of attribute "security.selinux" and element is "file".
2025-08-26 12:05:14,110 DEBUG [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification} core.Adapter - Adapter: [Notification] STAT: Pipeline finished processing message with messageId [testmessage0a809b34--54910c3d_198e5d6b5b0_-8000] exit-state [ERROR] started 2025-08-26 12:05:10.450 finished 2025-08-26 12:05:14.110 total duration: 3660 ms
2025-08-26 12:05:14,110 WARN  [http-nio-8080-exec-5] {mid=testmessage0a809b34--54910c3d_198e5d6b5b0_-8000, cid=Test a Pipeline 1, adapter=Notification} core.Adapter - Adapter [Notification] error processing message with ID [testmessage0a809b34--54910c3d_198e5d6b5b0_-8000]
org.frankframework.core.ListenerException: Pipe [putCanonicalNameInSessionKey] Exception on transforming input: Cannot transform input: (SAXParseException) line [1] column [284]: An invalid XML character (Unicode: 0x0) was found in the value of attribute "security.selinux" and element is "file".
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:621) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.core.Adapter.processMessageDirect(Adapter.java:540) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.management.bus.endpoints.TestPipeline.processMessage(TestPipeline.java:117) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.management.bus.endpoints.TestPipeline.runTestPipeline(TestPipeline.java:91) ~[frankframework-core-9.0.1.jar:9.0.1]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:268) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:263) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:196) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44) ~[spring-security-config-6.4.4.jar:6.4.4]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:268) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:263) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:196) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44) ~[spring-security-config-6.4.4.jar:6.4.4]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.frankframework.management.bus.endpoints.TestPipeline$$SpringCGLIB$$0.runTestPipeline(<generated>) ~[frankframework-core-9.0.1.jar:9.0.1]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:169) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.integration.handler.support.IntegrationInvocableHandlerMethod.doInvoke(IntegrationInvocableHandlerMethod.java:45) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.messaging.handler.invocation.InvocableHandlerMethod.invoke(InvocableHandlerMethod.java:119) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper$HandlerMethod.invoke(MessagingMethodInvokerHelper.java:1089) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.invokeHandlerMethod(MessagingMethodInvokerHelper.java:573) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.processInternal(MessagingMethodInvokerHelper.java:486) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.support.MessagingMethodInvokerHelper.process(MessagingMethodInvokerHelper.java:361) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.MethodInvokingMessageProcessor.processMessage(MethodInvokingMessageProcessor.java:114) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.ServiceActivatingHandler.handleRequestMessage(ServiceActivatingHandler.java:93) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractReplyProducingMessageHandler.handleMessageInternal(AbstractReplyProducingMessageHandler.java:146) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:105) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:73) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.MessageHandlerChain.lambda$configureChain$0(MessageHandlerChain.java:132) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:187) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:166) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:47) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.messaging.core.AbstractMessageSendingTemplate.send(AbstractMessageSendingTemplate.java:109) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.integration.handler.AbstractMessageProducingHandler.sendOutput(AbstractMessageProducingHandler.java:547) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageProducingHandler.doProduceOutput(AbstractMessageProducingHandler.java:361) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageProducingHandler.produceOutput(AbstractMessageProducingHandler.java:290) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageProducingHandler.sendOutputs(AbstractMessageProducingHandler.java:254) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractReplyProducingMessageHandler.handleMessageInternal(AbstractReplyProducingMessageHandler.java:152) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:105) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:73) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.MessageHandlerChain.handleMessageInternal(MessageHandlerChain.java:162) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageHandler.doHandleMessage(AbstractMessageHandler.java:105) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.handler.AbstractMessageHandler.handleMessage(AbstractMessageHandler.java:73) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.dispatcher.BroadcastingDispatcher.invokeHandler(BroadcastingDispatcher.java:235) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.dispatcher.BroadcastingDispatcher.dispatch(BroadcastingDispatcher.java:192) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.channel.AbstractSubscribableChannel.doSend(AbstractSubscribableChannel.java:72) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.channel.AbstractMessageChannel.sendInternal(AbstractMessageChannel.java:390) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.channel.AbstractMessageChannel.send(AbstractMessageChannel.java:334) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.messaging.core.GenericMessagingTemplate.doSend(GenericMessagingTemplate.java:187) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.messaging.core.GenericMessagingTemplate.doSendAndReceive(GenericMessagingTemplate.java:234) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.messaging.core.GenericMessagingTemplate.doSendAndReceive(GenericMessagingTemplate.java:47) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.messaging.core.AbstractMessagingTemplate.sendAndReceive(AbstractMessagingTemplate.java:46) ~[spring-messaging-6.2.2.jar:6.2.2]
	at org.springframework.integration.core.MessagingTemplate.sendAndReceive(MessagingTemplate.java:107) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.gateway.MessagingGatewaySupport.doSendAndReceive(MessagingGatewaySupport.java:687) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.gateway.MessagingGatewaySupport.sendAndReceive(MessagingGatewaySupport.java:615) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.springframework.integration.gateway.MessagingGatewaySupport.sendAndReceiveMessage(MessagingGatewaySupport.java:588) ~[spring-integration-core-6.4.3.jar:6.4.3]
	at org.frankframework.management.bus.LocalGateway.sendSyncMessage(LocalGateway.java:51) ~[frankframework-management-gateway-9.0.1.jar:9.0.1]
	at org.frankframework.console.controllers.FrankApiService.sendSyncMessage(FrankApiService.java:61) ~[frankframework-console-backend-9.0.1.jar:9.0.1]
	at org.frankframework.console.controllers.TestPipeline.testPipeLine(TestPipeline.java:103) ~[frankframework-console-backend-9.0.1.jar:9.0.1]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:359) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:196) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.proceed(AuthorizationManagerBeforeMethodInterceptor.java:268) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.attemptAuthorization(AuthorizationManagerBeforeMethodInterceptor.java:263) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.authorization.method.AuthorizationManagerBeforeMethodInterceptor.invoke(AuthorizationManagerBeforeMethodInterceptor.java:196) ~[spring-security-core-6.4.4.jar:6.4.4]
	at org.springframework.security.config.annotation.method.configuration.DeferringMethodInterceptor.invoke(DeferringMethodInterceptor.java:44) ~[spring-security-config-6.4.4.jar:6.4.4]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.frankframework.console.controllers.TestPipeline$$SpringCGLIB$$0.testPipeLine(<generated>) ~[frankframework-console-backend-9.0.1.jar:9.0.1]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.web.method.support.InvocableHandlerMethod.doInvoke(InvocableHandlerMethod.java:257) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.method.support.InvocableHandlerMethod.invokeForRequest(InvocableHandlerMethod.java:190) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.mvc.method.annotation.ServletInvocableHandlerMethod.invokeAndHandle(ServletInvocableHandlerMethod.java:118) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.invokeHandlerMethod(RequestMappingHandlerAdapter.java:986) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerAdapter.handleInternal(RequestMappingHandlerAdapter.java:891) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.mvc.method.AbstractHandlerMethodAdapter.handle(AbstractHandlerMethodAdapter.java:87) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1088) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.DispatcherServlet.doService(DispatcherServlet.java:978) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.FrameworkServlet.processRequest(FrameworkServlet.java:1014) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.FrameworkServlet.doPost(FrameworkServlet.java:914) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:590) ~[servlet-api.jar:6.0]
	at org.springframework.web.servlet.FrameworkServlet.service(FrameworkServlet.java:885) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at jakarta.servlet.http.HttpServlet.service(HttpServlet.java:658) ~[servlet-api.jar:6.0]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:195) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.frankframework.console.filters.SecurityLogFilter.doFilterInternal(SecurityLogFilter.java:44) ~[frankframework-console-backend-9.0.1.jar:9.0.1]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.2.2.jar:6.2.2]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.springframework.web.filter.ShallowEtagHeaderFilter.doFilterInternal(ShallowEtagHeaderFilter.java:107) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) ~[spring-web-6.2.2.jar:6.2.2]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.frankframework.console.filters.CacheControlFilter.doFilter(CacheControlFilter.java:50) ~[frankframework-console-backend-9.0.1.jar:9.0.1]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.frankframework.console.filters.CorsFilter.doFilter(CorsFilter.java:114) ~[frankframework-console-backend-9.0.1.jar:9.0.1]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.apache.tomcat.websocket.server.WsFilter.doFilter(WsFilter.java:51) ~[tomcat-websocket.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:108) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:231) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:365) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:101) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:91) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:85) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:179) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:107) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:93) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:75) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:101) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:110) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:233) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:186) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.servlet.handler.HandlerMappingIntrospector.lambda$createCacheFilter$3(HandlerMappingIntrospector.java:243) ~[spring-webmvc-6.2.2.jar:6.2.2]
	at org.springframework.web.filter.CompositeFilter$VirtualFilterChain.doFilter(CompositeFilter.java:113) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.filter.CompositeFilter.doFilter(CompositeFilter.java:74) ~[spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.config.annotation.web.configuration.WebMvcSecurityConfiguration$CompositeFilterChainProxy.doFilter(WebMvcSecurityConfiguration.java:238) ~[spring-security-config-6.4.4.jar:6.4.4]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) ~[catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) ~[catalina.jar:10.1.39]
	at org.springframework.security.web.FilterChainProxy.lambda$doFilterInternal$3(FilterChainProxy.java:231) ~[spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:365) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.access.intercept.AuthorizationFilter.doFilter(AuthorizationFilter.java:101) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:126) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:120) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:131) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:85) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:100) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:179) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:107) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:93) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.csrf.CsrfFilter.doFilterInternal(CsrfFilter.java:131) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.header.HeaderWriterFilter.doHeadersAfter(HeaderWriterFilter.java:90) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:75) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:82) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.context.SecurityContextHolderFilter.doFilter(SecurityContextHolderFilter.java:69) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.ForwardedHeaderFilter.doFilterInternal(ForwardedHeaderFilter.java:173) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:62) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.session.DisableEncodeUrlFilter.doFilterInternal(DisableEncodeUrlFilter.java:42) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:116) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:374) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:233) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:191) [spring-security-web-6.4.4.jar:6.4.4]
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:362) [spring-web-6.2.2.jar:6.2.2]
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:278) [spring-web-6.2.2.jar:6.2.2]
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:164) [catalina.jar:10.1.39]
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:140) [catalina.jar:10.1.39]
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:167) [catalina.jar:10.1.39]
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:90) [catalina.jar:10.1.39]
	at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:598) [catalina.jar:10.1.39]
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:115) [catalina.jar:10.1.39]
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:93) [catalina.jar:10.1.39]
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:74) [catalina.jar:10.1.39]
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:344) [catalina.jar:10.1.39]
	at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:397) [tomcat-coyote.jar:10.1.39]
	at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:63) [tomcat-coyote.jar:10.1.39]
	at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:905) [tomcat-coyote.jar:10.1.39]
	at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1743) [tomcat-coyote.jar:10.1.39]
	at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:52) [tomcat-coyote.jar:10.1.39]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1190) [tomcat-util.jar:10.1.39]
	at org.apache.tomcat.util.threads.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:659) [tomcat-util.jar:10.1.39]
	at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:63) [tomcat-util.jar:10.1.39]
	at java.base/java.lang.Thread.run(Thread.java:840) [?:?]
Caused by: org.frankframework.core.PipeRunException: Pipe [putCanonicalNameInSessionKey] Exception on transforming input: Cannot transform input: (SAXParseException) line [1] column [284]: An invalid XML character (Unicode: 0x0) was found in the value of attribute "security.selinux" and element is "file".
	at org.frankframework.pipes.XsltPipe.doPipe(XsltPipe.java:111) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.CorePipeProcessor.processPipe(CorePipeProcessor.java:36) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.TrackPreviousPipeInMetadataProcessor.processPipe(TrackPreviousPipeInMetadataProcessor.java:42) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.CheckMessageSizePipeProcessor.processPipe(CheckMessageSizePipeProcessor.java:37) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.LockerPipeProcessor.processPipe(LockerPipeProcessor.java:40) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.TransactionAttributePipeProcessor.execute(TransactionAttributePipeProcessor.java:97) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.TransactionAttributePipeProcessor.processPipe(TransactionAttributePipeProcessor.java:66) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.lambda$processPipe$0(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:46) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.AbstractPipeProcessor.processPipe(AbstractPipeProcessor.java:48) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.LimitingParallelExecutionPipeProcessor.processPipe(LimitingParallelExecutionPipeProcessor.java:68) ~[frankframework-core-9.0.1.jar:9.0.1]
	at jdk.internal.reflect.GeneratedMethodAccessor222.invoke(Unknown Source) ~[?:?]
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at 

...

jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.base/java.lang.reflect.Method.invoke(Method.java:569) ~[?:?]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethodWithGivenArgs(AbstractAspectJAdvice.java:638) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.aspectj.AbstractAspectJAdvice.invokeAdviceMethod(AbstractAspectJAdvice.java:628) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.aspectj.AspectJAroundAdvice.invoke(AspectJAroundAdvice.java:71) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:173) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.interceptor.ExposeInvocationInterceptor.invoke(ExposeInvocationInterceptor.java:97) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:184) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.springframework.aop.framework.CglibAopProxy$DynamicAdvisedInterceptor.intercept(CglibAopProxy.java:727) ~[spring-aop-6.2.2.jar:6.2.2]
	at org.frankframework.processors.CorePipeLineProcessor$$SpringCGLIB$$0.processPipeLine(<generated>) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.LockerPipeLineProcessor.processPipeLine(LockerPipeLineProcessor.java:60) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.TransactionAttributePipeLineProcessor.processPipeLine(TransactionAttributePipeLineProcessor.java:49) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.LimitingParallelExecutionPipeLineProcessor.processPipeLine(LimitingParallelExecutionPipeLineProcessor.java:42) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.CachePipeLineProcessor.processPipeLine(CachePipeLineProcessor.java:41) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.processors.InputOutputPipeLineProcessor.processPipeLine(InputOutputPipeLineProcessor.java:44) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.core.PipeLine.process(PipeLine.java:413) ~[frankframework-core-9.0.1.jar:9.0.1]
	at org.frankframework.core.Adapter.processMessageWithExceptions(Adapter.java:613) ~[frankframework-core-9.0.1.jar:9.0.1]
	... 221 more
2025-08-26 12:05:14,125 DEBUG [http-nio-8080-exec-5] {} stream.Message - returning String Message[5bc0ed01:String] as InputStream
2025-08-26 12:05:14,125 DEBUG [http-nio-8080-exec-5] {} core.PipeLineSession - Closing PipeLineSession
```

### Anything else?

_No response_

--- a/aws/src/main/java/org/frankframework/filesystem/AmazonS3FileSystem.java
+++ b/aws/src/main/java/org/frankframework/filesystem/AmazonS3FileSystem.java
@@ -555,11 +555,12 @@ public class AmazonS3FileSystem extends AbstractFileSystem<S3FileRef> implements
 	public long getFileSize(S3FileRef f) throws FileSystemException {
 		// Probably a folder, which cannot have a size.
 		if (!f.hasName()) {
-			return 0;
+			return 0L;
 		}
 
 		updateFileAttributes(f);
-		return f.getContentLength();
+		Long contentLength = f.getContentLength();
+		return contentLength != null ? contentLength : 0L;
 	}
 
 	@Override

--- a/core/src/main/java/org/frankframework/util/MessageUtils.java
+++ b/core/src/main/java/org/frankframework/util/MessageUtils.java
@@ -64,9 +64,9 @@ import org.frankframework.stream.MessageContext;
 public class MessageUtils {
 
 	private static final Logger LOG = LogUtil.getLogger(MessageUtils.class);
-	private static final int CHARSET_CONFIDENCE_LEVEL = AppConstants.getInstance().getInt("charset.confidenceLevel", 65);
 	private static final Tika TIKA = new Tika();
 
+	public static final int CHARSET_CONFIDENCE_LEVEL = AppConstants.getInstance().getInt("charset.confidenceLevel", 65);
 	public static final String JSON_TEMPLATE_VALUE_QUOTED = "{\"%s\": \"%s\"}";
 	public static final String JSON_TEMPLATE_VALUE_UNQUOTED = "{\"%s\": %s}";
 	public static final String DEFAULT_MESSAGE_ID_PREFIX = "FFMSG";

--- a/filesystem/src/main/java/org/frankframework/filesystem/LocalFileSystem.java
+++ b/filesystem/src/main/java/org/frankframework/filesystem/LocalFileSystem.java
@@ -43,6 +43,9 @@ import jakarta.annotation.Nullable;
 import org.apache.commons.lang3.StringUtils;
 import org.apache.commons.lang3.exception.ExceptionUtils;
 
+import com.ibm.icu.text.CharsetDetector;
+import com.ibm.icu.text.CharsetMatch;
+
 import lombok.Getter;
 import lombok.extern.log4j.Log4j2;
 
@@ -50,6 +53,8 @@ import org.frankframework.configuration.ConfigurationException;
 import org.frankframework.doc.Default;
 import org.frankframework.stream.Message;
 import org.frankframework.stream.PathMessage;
+import org.frankframework.util.MessageUtils;
+import org.frankframework.util.XmlEncodingUtils;
 
 /**
  * {@link IWritableFileSystem FileSystem} representation of the local filesystem.
@@ -188,10 +193,10 @@ public class LocalFileSystem extends AbstractFileSystem<Path> implements IWritab
 		return userDefinedAttributes.list().stream().anyMatch(attributeName::equals);
 	}
 
-	private static @Nonnull String readAttribute(@Nonnull UserDefinedFileAttributeView userDefinedAttributes, @Nonnull String attributeName) throws IOException {
+	private @Nonnull String readAttribute(@Nonnull Path file, @Nonnull UserDefinedFileAttributeView userDefinedAttributes, @Nonnull String attributeName) throws IOException {
 		ByteBuffer bfr = ByteBuffer.allocate(userDefinedAttributes.size(attributeName));
 		userDefinedAttributes.read(attributeName, bfr);
-		return new String(bfr.array());
+		return readAttributeValue(file, attributeName, bfr);
 	}
 
 	@Override
@@ -369,7 +374,7 @@ public class LocalFileSystem extends AbstractFileSystem<Path> implements IWritab
 			if (Files.getFileStore(f).supportsFileAttributeView(UserDefinedFileAttributeView.class)) {
 				UserDefinedFileAttributeView userDefinedAttributes = getUserDefinedAttributes(f);
 				if (hasAttribute(userDefinedAttributes, ORIGINAL_LAST_MODIFIED_TIME_ATTRIBUTE)) {
-					String lastModifiedTime = readAttribute(userDefinedAttributes, ORIGINAL_LAST_MODIFIED_TIME_ATTRIBUTE);
+					String lastModifiedTime = readAttribute(f, userDefinedAttributes, ORIGINAL_LAST_MODIFIED_TIME_ATTRIBUTE);
 					return new Date(Long.parseLong(lastModifiedTime));
 				}
 			}
@@ -391,7 +396,7 @@ public class LocalFileSystem extends AbstractFileSystem<Path> implements IWritab
 			String attrSpec = "user:" + String.join(",", attributeNames);
 			Map<String, Object> result = new LinkedHashMap<>();
 			Files.readAttributes(file, attrSpec)
-					.forEach((name, value) -> result.put(name, readAttributeValue(value)));
+					.forEach((name, value) -> result.put(name, readAttributeValue(file, name, value)));
 			return result;
 		} catch (UnsupportedOperationException e) {
 			return null;
@@ -400,14 +405,34 @@ public class LocalFileSystem extends AbstractFileSystem<Path> implements IWritab
 		}
 	}
 
-	private String readAttributeValue(Object attributeValue) {
-		if (attributeValue instanceof byte[] bytes) {
-			return new String(bytes);
-		} else if (attributeValue instanceof ByteBuffer buffer) {
-			return new String(buffer.array());
-		} else {
-			return attributeValue.toString();
+	private @Nonnull String readAttributeValue(@Nonnull Path file, @Nonnull String name, @Nonnull Object attributeValue) {
+		try {
+			if (attributeValue instanceof byte[] bytes) {
+				return bytesToString(bytes);
+			} else if (attributeValue instanceof ByteBuffer buffer) {
+				return bytesToString(buffer.array());
+			} else {
+				return attributeValue.toString();
+			}
+		} catch (Exception e) {
+			log.warn(() -> "Error parsing file attribute [%s] on file [%s]:".formatted(name, file), e);
+			return "<attribute not readable>";
 		}
+	}
+
+	@Nonnull
+	private static String bytesToString(@Nonnull byte[] bytes) {
+		CharsetDetector detector = new CharsetDetector();
+		detector.setText(bytes);
+		CharsetMatch match = detector.detect();
+		Charset charset;
+		if (match.getConfidence() >= MessageUtils.CHARSET_CONFIDENCE_LEVEL) {
+			charset = Charset.forName(match.getName());
+		} else {
+			// If confidence was low, use system default charset
+			charset = Charset.defaultCharset();
+		}
+		return XmlEncodingUtils.replaceNonValidXmlCharacters(new String(bytes, charset), '?', false, true);
 	}
 
 	@Nullable

--- a/filesystem/src/test/java/org/frankframework/filesystem/LocalFileSystemTest.java
+++ b/filesystem/src/test/java/org/frankframework/filesystem/LocalFileSystemTest.java
@@ -5,11 +5,16 @@ import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
+import java.io.ByteArrayInputStream;
 import java.io.OutputStream;
 import java.io.PrintWriter;
 import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
+import java.nio.file.attribute.UserDefinedFileAttributeView;
+import java.util.Map;
 
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.io.TempDir;
@@ -117,4 +122,34 @@ public class LocalFileSystemTest extends FileSystemTest<Path, LocalFileSystem> {
 		}
 	}
 
+	@Test
+	void localFileSystemTestReadExtendedAttributeWithIllegalEncodedValue() throws Exception {
+		String filename = "createFileAbsolute" + FILE1;
+		String contents = "regeltje tekst";
+
+		fileSystem.configure();
+		fileSystem.open();
+
+		deleteFile(null, filename);
+		waitForActionToFinish();
+
+		Path file = fileSystem.toFile(fileSystem.getRoot() + "/" + filename);
+		fileSystem.createFile(file, new ByteArrayInputStream(contents.getBytes()));
+
+		waitForActionToFinish();
+
+		ByteBuffer bfr = ByteBuffer.allocate(10);
+		bfr.put("test".getBytes());
+		bfr.put((byte)0);
+		bfr.put("test".getBytes());
+
+		UserDefinedFileAttributeView fileAttributeView = Files.getFileAttributeView(file, UserDefinedFileAttributeView.class);
+		fileAttributeView.write("testAttribute", bfr);
+
+		// test
+		Map<String, Object> additionalFileProperties = fileSystem.getAdditionalFileProperties(file);
+
+		assertTrue(additionalFileProperties.containsKey("testAttribute"), "Custom file attribute 'testAttribute' missing");
+		assertEquals("?", additionalFileProperties.get("testAttribute"));
+	}
 }

